### ISSUE TITLE
Single product page has changed

### ISSUE THREAD

**Issue by ab-marketing** [woocommerce/storefront#1129]
Hi Storefront!

I think something goes wrong with the single product pages, where the sticky add to cart is activated. The page looks like very different since 2.5.0. update. Here is a live demo: https://moksaiskola.hu/_teszt_ilonamed/hu/shop/4-2-multipack-6-db-moksa-csepp-6x10ml/ I just tested it with our dev site and we must skip this update to waiting fix this issue.

It should look like as this: https://ilonamed.at/hu/termek/4-2-multipack-6-db-moksa-csepp-6x10ml/

The W3C validator gives some errors too. If i checked this as false, than i must say sorry and i will digging into the child theme; but i think it would be an official error.

Otherwise many thanks for this nice theme! Have a nice day.
Matthew

**Comment by tiagonoronha** [comment]
@ab-marketing there's CSS in that site that is causing the issue:

<img width="438" alt="Screenshot 2019-05-10 at 09 28 01" src="https://user-images.githubusercontent.com/1177726/57513370-fa20e000-7305-11e9-84ec-dc1a6ab5777b.png">

That CSS is not part of Storefront. Please check your plugins and customizations.

**Comment by ab-marketing** [comment]
Hi tiagonoronha!

I found the bug. Now our child theme is working correctly with WP 5.1.1. but isn't working with WP 5.2. !!!

I just digged the problem out; and it is look like something wrong with the new WP core to our theme. It is interesting; that i don't get the error with the original storefront theme, i must check what is wrong with my child theme. 

Sorry for my false report. Have a nice day!
Bye
